keyboards/J80/J80.h

```diff
@@ -0,0 +1,79 @@
+/* Copyright 2019 JER
```

Comment:
This file and the J80 folder are still uppercase :)

Reply:
Renamed J80.h to j80.h. Tried renaming the folder on both my local clone and on my fork, but when looking at the "Changed files" tab the folder name is still uppercase.